tutorons/python/explain.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env python
+# encoding: utf-8
+
+from __future__ import unicode_literals
+import logging
+from bs4 import BeautifulSoup
+import collections
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+fcns = open('tutorons/python/fcns.txt', 'r').read()
+builtin_doc = BeautifulSoup(fcns)
+dt = builtin_doc.findAll('dt')  # headers
+dd = builtin_doc.findAll('dd')  # explanations
+explanations = collections.OrderedDict()  # builtin : explanation (dt, dd)
```

Comment:
Seems a little strange to be looking this stuff up from an HTML document, when all we want is a lookup table from a plaintext key to a UTF-8 string.  Could we just store this in a JSON file or a separate Python module, so we can read it in as what it is---a lookup table?